repos/system_upgrade/el7toel8/actors/setuptargetrepos/actor.py

```diff
@@ -59,7 +77,8 @@ def process(self):
                 for repo in repo_file.data:
                     enabled_repos.add(repo.repoid)
 
-        skipped_repos = skipped_repos.intersection(set(used.keys()))
+        # TODO(pstodulk): the part of the workaround mentioned above..
+        skipped_repos = skipped_repos.intersection(set(used.keys())) - EXTRAS_REPOS
```

Comment:
@pirat89 we have here "mapped_repos", "used_repos", "rhel_repos", and "skipped_repos", I think it deserves some commentary :) It can wait but I guess potential reviewers can benefit from it in the future...

Reply:
Yeah. I spend a time checking all "tentacles" :) This actor is one of those that are expected to be rewritten in future when there is a time :) after the `targetuserspacecreator` is rewritten first.